ui/imports/shared/status/StatusSettingsLineButton.qml

```diff
@@ -113,13 +124,19 @@ Rectangle {
 
     MouseArea {
         anchors.fill: parent
-        enabled: root.isEnabled
         hoverEnabled: true
         onEntered: root.isHovered = true
         onExited: root.isHovered = false
         onClicked: {
-            root.clicked(!root.switchChecked)
+            if (root.isEnabled) {
+                root.clicked(!root.switchChecked)
+            }
         }
         cursorShape: Qt.PointingHandCursor
+
+        StatusToolTip {
+            text: root.toolTipText
+            visible: !root.isEnabled && root.isHovered && d.isToolTipTextPresent(toolTipText)
```

Comment:
is `isToolTipTextPresent` really needed? It's used just in a single place where it's properly set 🙂